circle.yml

```diff
@@ -58,7 +58,7 @@ version: 2
 jobs:
   generic:
     docker:
-      - image: balenalib/amd64-alpine-node:10
+      - image: balenalib/amd64-alpine-node:12
```

Comment:
Not strictly necessary as it turns out but it makes sense to run the tests on node 12 since that's what we run on

Reply:
When I first started using supervisor I had some issues when running on node 10 so I think this is a good change.